### PR TITLE
docs: add opendal services

### PR DESCRIPTION
**Pull request by simonsan**
Add basic service documentation for using opendal.

Ref.: https://github.com/rustic-rs/rustic/issues/1278

### CODE REVIEW

File: src/commands/init/rclone.md [hunk 1]
```diff
@@ -44,9 +44,9 @@ $ rclone ls b2prod:yggdrasil/foo/bar/baz
     448 bar/baz/keys/4bf9c78049de689d73a56ed0546f83b8416795295cda12ec7fb9465af3900b44
 ```
 
-Rclone can be `configured with environment variables`_, so for instance
-configuring a bandwidth limit for rclone can be achieved by setting the
-`RCLONE_BWLIMIT` environment variable:
+Rclone can be configured with environment variables prefixed by `RCLONE_`, so
+for instance configuring a bandwidth limit for rclone can be achieved by setting
+the `RCLONE_BWLIMIT` environment variable:
```
Comment (simonsan) [inline src/commands/init/rclone.md:49]: We could also talk about the option to set environment variables in the config file here, actually?

Comment (aawsome) [reply src/commands/init/rclone.md:49]: I added this.

**Review by aawsome — APPROVED** [review]
LGTM

**Comment by bebehei** [comment]
Total respect. Thanks for fixing this so fast.

**Pull request merged**